ibis/expr/tests/conftest.py

```diff
@@ -98,10 +103,29 @@ def lineitem(con):
 
 
 @pytest.fixture
-def geo_con():
-    return GeoMockConnection()
-
-
-@pytest.fixture
-def geo_table(geo_con):
-    return geo_con.table('geo')
+def geo_proxy():
+    return {
+        'postgis': GeoMockConnectionPostGIS(),
+        'omniscidb': GeoMockConnectionOmniSciDB(),
+    }
+
+
+@pytest.hookimpl(hookwrapper=True)
+def pytest_pyfunc_call(pyfuncitem):
```

Comment:
Why does this need to be duplicated here?

Reply:
it seems the pytest_pyfunc_call from tests/all conftest didn't work for expr tests .. 
xfail_unsupported wasn't working ...
so I just created a simplified version of the original one.

Reply:
Can you import the one from `tests/all` here?